server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -288,6 +295,10 @@ TransportResponse handleExtensionRequest(ExtensionRequest extensionRequest) thro
         } else if (extensionRequest.getRequestType() == RequestType.REQUEST_EXTENSION_CLUSTER_SETTINGS) {
             ClusterSettingsResponse clusterSettingsResponse = new ClusterSettingsResponse(clusterService);
             return clusterSettingsResponse;
+        } else if (extensionRequest.getRequestType() == RequestType.REQUEST_ACTION_LISTENER_ON_FAILURE) {
```

Comment:
Since we're adding more extension request handlers, I would suggest we refactor this method to use a switch statement rather than adding all these `else if` statements